gateways/mercadopago/accesstoken/accesstoken.go

```diff
@@ -0,0 +1,91 @@
+package accesstoken
```

Comment:
What you think to use only one package (`mercadorpago`) to import like `paygo/gateways/mercadopago/config` and `paygo/gateways/mercadopago/accesstoken`?

Following your abordage when import this gateway we will import as `paygo/gateways/mercadopago/accesstoken/accesstoken`

Reply:
No, man, this way the imports will look like `paygo/gateways/mercadopago/accesstoken`, the files doesn't matter to do import the packages, I guess. Whatever when I imported some packages, they was look like this way.